### ISSUE TITLE
cpPosition fails with material design

### ISSUE THREAD

**Issue by MurphysChaos** [zefoy/ngx-color-picker#56]
When using the material design library for Angular, the position of the color picker pop-up is in the top left corner (default CSS position). Material design add some classes and modifies the _innerHTML_ of the target box as follows:

### With _md-button_ directive
```
<button _ngcontent-c0="" class="cp-button mat-button" md-button=""
    ng-reflect-color-picker="#00ff00" ng-reflect-cp-position="bottom" 
    ng-reflect-cp-position-offset="50%" ng-reflect-cp-position-relative-to-arrow="true">
    <span class="mat-button-wrapper">
        This is a button
    </span>
    <div class="mat-button-ripple mat-ripple" md-ripple=""
        ng-reflect-trigger="[object HTMLButtonElement]" ng-reflect-centered="false"
        ng-reflect-disabled="false"></div>
    <div class="mat-button-focus-overlay">
    </div>
</button>
```

### Normal _colorPicker_ HTML
```
<button _ngcontent-c0="" class="cp-button" ng-reflect-color-picker="#00ff00"
    ng-reflect-cp-position="bottom" ng-reflect-cp-position-offset="50%"
    ng-reflect-cp-position-relative-to-arrow="true">
    This is a button
</button>
```

**Comment by sconix** [comment]
You should be able to tweak the positioning with CSS, for example using transform or margin. I will check it tomorrow if there is some tweaks that I could do within the library that it plays nice with material components.

**Comment by sconix** [comment]
There seems to be a bug in the new position code when used with buttons. I will release a fix soon.

**Comment by sconix** [comment]
This should be now fixed in 4.3.3 release.
